src/main/resources/api/openapi.yaml

```diff
@@ -2129,6 +2146,38 @@ paths:
               schema:
                 $ref: '#/components/schemas/ApiError'
 
+  /blocks/{headerId}/proofFor/{txId}:
```

Comment:
Is it possible to get proof by using `txId` alone? This would simplify development of applications.  txId is unique and in most cases it is enough to refer to the specific transaction, but not here. Thus every future application will have to solve the same problem of finding headerId for a given txId.

Reply:
@aslesarenko it is not possible as the node is not storing transactions index, and no plans to add it for minimal settings.